src/qbr.py

```diff
@@ -5,62 +5,70 @@
 # Created       : Tue, 26 Jan 2016
 # Last Modified : Sun, 31 Jan 2016
 
+# Built-ins
+from sys import exit as die
+import argparse
+from typing import Union, List
 
-from sys import exit as Die
+# 3rd Party
 try:
-    import sys
-    import kociemba
-    import argparse
-
-    from combiner import combine
-    from video import webcam
-    from normalizer import normalize
+    import kociemba     # type: ignore
```

Comment:
Can `# type: ignore` be removed? If not, please remove leading spaces in between `np_ndarray` and `# type: ignore`